compiler/cpp/src/generate/t_cpp_generator.cc

```diff
@@ -116,7 +116,8 @@ class t_cpp_generator : public t_oop_generator {
                                    bool read = true,
                                    bool write = true,
                                    bool swap = false,
-                                   bool stream = false);
+                                   bool stream = false,
+                                   bool is_struct = false);
```

Comment:
is this necessary? it's either struct or exception, and there is already bool for that. or am I missing something? IMHO there is too many of those bools already :)